.gitignore

```diff
@@ -58,6 +58,7 @@ build/
 \#*
 \#*\#
 .#*
+rustc-ice-*.txt
```

Comment:
I don't like this change. Now my folders fill up with ICE files and I don't even notice, until I do `ls` and I drown in a sea of files.